FSI/flap_perp/SU2-FEniCS/Output/plotDisplacement.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+gnuplot -p << EOF
+set grid
+set title 'Displacement of the Flap Tip'
+set xlabel 'Time [s]'
+set ylabel 'X-Displacement [m]'
+plot "point1.watchpoint.txt" using 1:5 smooth cumulative title ""
```

Comment:
This name has changed in recent preCICE versions:
```suggestion
plot "precice-Calculix-watchpoint-point1.log using 1:5 smooth cumulative title ""
```
we need to update it elsewhere as well. See changes proposed in https://github.com/precice/tutorials/pull/100.